lib/regular-update.js

```diff
@@ -4,7 +4,7 @@ const request = require('request');
 
 // Map from URL to { timestamp: last fetch time, interval: in milliseconds,
 // data: data }.
-let regularUpdateCache = Object.create(null);
+let regularUpdateCache = new Map();
```

Comment:
All `regularUpdateCache[url]` need to be changed to either `regularUpdateCache.get(url)` or `regularUpdateCache.set(url, value)` in order to use `Map`

Reply:
I'm going to reset these changes.